src/test/java/se/liu/ida/hefquin/engine/data/mappings/impl/EntityMappingImplTest.java

```diff
@@ -0,0 +1,79 @@
+package se.liu.ida.hefquin.engine.data.mappings.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.graph.NodeFactory;
+import org.apache.jena.sparql.core.Var;
+import org.junit.Test;
+
+import se.liu.ida.hefquin.engine.data.SolutionMapping;
+import se.liu.ida.hefquin.engine.data.mappings.EntityMapping;
+import se.liu.ida.hefquin.engine.data.utils.SolutionMappingUtils;
+
+public class EntityMappingImplTest {
+	@Test
+	public void TranslateCartesianProduct() {

```

Comment:
Function names should always start with a lower case letter (i.e., `translateCartesianProduct()` instead of `TranslateCartesianProduct()`). Additionally, for functions that represent unit tests that test a specific function of the corresponding class (such as the `applyToSolutionMapping` function in this case), the name of the unit test function should reflect what function is tested.
```suggestion
	public void applyToSolutionMappingTest() {
```